### ISSUE TITLE
Classes for everything

### ISSUE THREAD

**Issue by sajidali007522** [sajidali007522/edu#2]
You said everything like colors, font-weight, etc need to hanlde by class like "has_border_color_secondary"
If we use this structure then heading tags contain properties like, font-weight, color, font-size etc. so in one html tag will contain list of properties and it's not good and not readable. 

I will create all classes you want so you can use during working, so right now you can use my html or can use classes also. is this ok for you?

**Comment by hone-zaun** [comment]
Yes we will have HTML-tags like `<h1 class="has_font_primary has-text-weight-bold has_color_secondary is-uppercase">some heading</h1>`. And we need it this way because we want to be able to easily customize font-family, colors and so on.

It is much more complicated for our usecase to change colors and fonts in a css-stylesheet when there are several tags/classes to change.
